### ISSUE TITLE
Delete old feature branches at https://github.com/18F/calc

### ISSUE THREAD

**Issue by mshenfield** [18F/calc#253]
Hey! I was interested in contributing to this project. I appreciate the awesome service this provides.

There are several feature branches that don't seem to correspond to specific issues, or cards on the [Trello board](https://trello.com/b/LjXJaVbZ/calc-project-board), and haven't been updated in 3+ months. This makes it feel a little perilous for new users to make changes, and goes against one of the recommendations outlined in the [internal contribution guide](https://github.com/18F/calc/blob/master/CONTRIBUTING-internal.md#git-workflow).

You can run the command

```
for k in `git branch -r | perl -pe 's/^..(.*?)( ->.*)?$/\1/'`; do echo -e `git show --pretty=format:"%Cgreen%ci %Cblue%cr%Creset" $k -- | head -n 1`\\t$k; done | sort -r
```

to see the current list of remote branches and their last commit date ([thanks VonC](http://stackoverflow.com/questions/2514172/listing-each-branch-and-its-last-revisions-date-in-git)).

Here's the output when I run it:

```
2015-10-13 15:09:15 -0700 2 hours ago	origin/master
2015-10-13 15:09:15 -0700 2 hours ago	origin/HEAD
2015-09-25 11:59:29 -0700 3 weeks ago	origin/aug-data
2015-08-25 20:48:43 +0000 7 weeks ago	origin/selenium-fix
2015-08-17 17:16:01 +0000 8 weeks ago	origin/new-search-features
2015-08-04 11:32:09 -0700 2 months ago	origin/update-description
2015-07-31 19:10:33 +0000 2 months ago	origin/fix-radio-buttons
2015-07-30 20:23:22 +0000 3 months ago	origin/add-year-out
2015-07-16 19:47:29 +0000 3 months ago	origin/public-docs
2015-07-14 14:55:41 +0000 3 months ago	origin/indexes
2015-07-13 19:54:41 +0000 3 months ago	origin/graph-img
2015-07-09 13:55:32 -0700 3 months ago	origin/add-data-readme
2015-07-09 13:15:50 -0700 3 months ago	origin/update-deploy-docs
2015-07-08 14:56:49 +0000 3 months ago	origin/selenium
2015-07-07 09:36:17 -0700 3 months ago	origin/decimal
2015-06-24 19:39:31 +0000 4 months ago	origin/fix-tests
2015-06-23 12:24:46 -0400 4 months ago	origin/docs
2015-06-23 11:20:49 -0700 4 months ago	origin/experience-range-query-string
2015-06-22 18:22:37 +0000 4 months ago	origin/education-endpoint
2015-06-18 20:43:10 +0000 4 months ago	origin/cf-migrations
2015-06-02 14:54:06 +0000 4 months ago	origin/standard-deviation
2015-05-12 09:09:12 -0400 5 months ago	origin/about-tweaks
2015-04-29 12:32:51 -0700 6 months ago	origin/add-circle
2015-01-27 11:12:09 -0800 9 months ago	origin/cloudfoundry
```

**Comment by brethauer** [comment]
Hey @mshenfield! Thanks for posting this issue, and sorry for the delay in getting back to you.  We've been paused for a bit.  Great suggestion and we'll def do it when we can spin back up!

**Comment by mshenfield** [comment]
No worries! Definitely a little nitpicky, thanks for being so receptive.

**Comment by Connery-LCSolutions** [comment]
Hi All,
     New to the calc repo. Is there a tech stack list and UML/visio diagram
posted? I see that it's a Django project, but not much else. Sorry and
thank you!

Very Respectfully,

Daniel Connery

On Sat, Oct 24, 2015 at 8:03 AM, Max Shenfield <notifications@github.com>
wrote:

> No worries! Definitely a little nitpicky, thanks for being so receptive.
>
> On Fri, Oct 23, 2015 at 10:52 PM, Nick Brethauer <notifications@github.com
> >
> wrote:
>
> > Hey @mshenfield <https://github.com/mshenfield>! Thanks for posting this
> > issue, and sorry for the delay in getting back to you. We've been paused
> > for a bit. Great suggestion and we'll def do it when we can spin back up!
> >
> > —
> > Reply to this email directly or view it on GitHub
> > <https://github.com/18F/calc/issues/253#issuecomment-150749787>.
>
> >
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/18F/calc/issues/253#issuecomment-150803034>.
>



-- 
Daniel Connery
Executive Partner
Life Cycle Solutions
O: 703-647-4711
M: 214-934-3025

**Comment by toolness** [comment]
Hey, sorry it's taken us so long to get around to this--the project was unmaintained for several months but it looks like it might be spun-up again, so I'm going through the backlog of existing issues and found this one.

I've just cleaned out all the really old branches, and now there are only a handful of unmerged ones left that we need to learn more about. I've created WIP PRs for those, so I'm closing this issue for now.

Thanks for reporting it @mshenfield!
